src/data/socialwidgets.json

```diff
@@ -26,6 +26,26 @@
             "type": 0
         }
     },
+    "Disqus": {
+        "domains": [
+            "disqus.com",
```

Comment:
I think I see what you're getting at here. Mind taking a look at the latest commit to see that I'm understanding you correctly?

I changed the `slice()` method to call it at `.slice(2)` instead though, so that it chops off both `*` and the `.` characters from the domain